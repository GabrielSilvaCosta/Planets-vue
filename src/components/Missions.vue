<template>
  <div class="missions">
    <Title headline="Missoes" />
  </div>
</template>

<script>
import Title from "./Title.vue";
export default {
  name: "Missions",
  components: {
    Title,
  },
};
</script>

<style scoped></style>
