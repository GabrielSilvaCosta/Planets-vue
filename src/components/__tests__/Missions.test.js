import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";
import Missions from "../Missions.vue";

describe("Missions", () => {
    it("should render correctly", () => {
        const wrapper = mount(Missions, {
            props: {
                missions: [],
            },
        });
        expect(wrapper.html()).toMatchSnapshot();
    });
})