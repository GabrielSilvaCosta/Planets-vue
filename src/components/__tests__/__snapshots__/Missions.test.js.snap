// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Missions > should render correctly 1`] = `
"<div class="missions" missions="">
  <div data-v-6db1a9b7="" class="title">
    <h1 data-v-6db1a9b7="">Missoes</h1>
  </div>
</div>"
`;
